railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -205,6 +211,7 @@ def create_root_files
         build(:configru)
         build(:gitignore)   unless options[:skip_git]
         build(:gemfile)     unless options[:skip_gemfile]
+        build(:version_control)
```

Comment:
Why not check the `options[:skip_git]` flag here like how its been done for `:gitignore`?